src/RefractSourceMap.h

```diff
@@ -0,0 +1,73 @@
+//
+//  RefractSourceMap.h
+//  drafter
+//
+//  Created by Jiri Kratochvil on 31/07/15.
+//  Copyright (c) 2015 Apiary Inc. All rights reserved.
+//
+
+#ifndef DRAFTER_REFRACTSOURCEMAP_H
+#define DRAFTER_REFRACTSOURCEMAP_H
+
+#include "Serialize.h"
+
+namespace drafter {
+
+
+    refract::IElement* BytesRangeToRefract(const mdp::BytesRange& bytesRange);
+
+    template<typename T>
+    refract::IElement* SourceMapToRefract(const T& sourceMap) 
```

Comment:
All non-template functions (in this caseBytesRangeToRefract()). 
All others are template functions. If you want them get into `.cc` file so please write there prefered solution. You can not "just" put them into `.cc` until C++11 

Try it yourself!